InstrumentationEngine.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28727.205
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32201.480
```

Comment:
Is this intended? I found that it was not possible to update to vs2022 without also updating extensions.base since vs 2022 does not want to build anything below 4.5.2.

We also haven't released 17.2 yet publicly

Reply:
This version header gets updated automatically whenever a new major VS version builds the solution and doesn't actually do anything besides update the icon for the .sln file.

Spoke offline, we'll continue to expect support on VS2019 (our build pools won't change) and will make a larger move to VS2022 separately.